internal/testing/testdata/testdata.go

```diff
@@ -317,6 +317,14 @@ var (
 		Justification: "Derived from SPDX DEPENDS_ON relationship",
 	}
 
+	isOccJustifyFile = &model.IsOccurrenceSpecInputSpec{
```

Comment:
not sure why it's not erroring but this is now `IsOccurrenceInputSpec`

Reply:
prob cause its not updated and git says it can automerge, rebased and updated the ref!

Reply:
resolving this, passing tests with rebase